LoopUI/Views/DeviceStatusHUDView.swift

```diff
@@ -53,6 +53,22 @@ import LoopKitUI
         }
     }
     
+    public var adjustViewsForNarrowDisplay: Bool = false {
+        didSet {
+            if adjustViewsForNarrowDisplay != oldValue {
```

Comment:
I think this logic looks fine; but I think the evaluation should be done based on the provided space (i.e. the view's bounds), rather than the device size.

Reply:
will update to use the status VCs bounds width.